nvidia/glvndr.go

```diff
@@ -0,0 +1,35 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2024 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package nvidia
+
+// LibGlvndGlobs are related to libglvndr library.
+//
+// The library is further described at https://github.com/NVIDIA/libglvnd.
```

Comment:
Should we mention that this should match content of `cmd/snap-confine/mount-support-nvidia.c`  `glvnd_globs`? Should we perhaps test for this?

Reply:
I want to remove the other side rather than document it in depth.

Reply:
The other side is gone now.